walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -164,176 +121,141 @@ After the mobile app is registered, it is displayed on the console.
-. Each edit will create new Task
-. We can review all offline items directly by clicking on `Offline Changes` button.
-. Please restore connectivity (depending on your device)
-. We should see all `Offline Changes` reflected back to server and appearing in second instance of the application.
-
-[time=6]
-== Conflict resolution functionalities
-
-Data Conflicts can happen when resource we modified was also modified on server.
-DataSync Framework enables multiple strategies to deal with conflicts. 
-By default Showcase will resolve conflicts by applying all server side changes on top of the client.
-In case of direct conflicts client field value will be used. 
-To simulate conflict we need to go offline and make edits in two instance of the application at the same time.
+. Go to the link:{route-mdc-server-host}[Mobile Developer Console, window="_blank"].
+. Click on the *showcase-app* card to display the configuration screen.
+. Click the *Mobile Services* tab.
+. Click *Bind to App* for *Identity Management* in the *Unbound Services* section, with the following parameters:
+.. Admin User Name: `admin`
+.. Admin User Password: `admin`
+.. Client Type: `public`
+. Close the wizard. Once the binding process is completed, you should the *Identity Management* row is moved to the `Bound Services` section.
+. Go back to the `Configuration` tab of the app and you should see the content of `mobile-services.json` is updated. Copy the content and save it somewhere on the computer.
+
+=== Add Identity Management service to the data sync server
+
+. Go to the `Mobile Services` tab again and expand the *Identity Management* row. Open the `Keycloak Realm URL` in a new tab.
+. Login with username `admin` and password `admin`.
+. After the console is loaded, we need to create a new client for the showcase data sync server and add some users for the showcase app:
+.. Choose `Configure -> clients` in the menu.
+.. Click on the `Create` button on the top-right corner, and create a new client with the following parameters:
+... Client ID: showcase-app-server
+... Do not change the rest of the fields.
+.. Save the client. You should see the *Settings* page of the client.
+.. Change the `Access Type` to `bearer-only` and save again.
+.. Click on the `Installation` tab, and select `Keycloak OIDC JSON` format. Use the `Download` button to save the configuration file to a directory on the computer.
+.. Select `Users` on the left menu, and click on `View all users`. You will see there is only 1 admin user in the realm. Click on `Add user` to create new ones. Pick a username you like and save.
+.. Go to `Credentials` tab and set a password for this user. Set `Temporary` option to `OFF`.
+. Next, we need to update the showcase sync server app to use the downloaded configuration file:
+.. Go to the link:{openshift-host}/console/project/{walkthrough-namespace}/config-maps[Config Maps page of the OpenShift console], and click on `Create Config Map`.
```

Comment:
I think command line path for creating configmap from file will me much simpler TBH. 
`oc create configmap  showcase-server-idm-config --from-file=keycloak.json`

Especially when people will already have oc client pointed to that namespace

Reply:
yes you are right, but I tried to avoid using CLIs in this walkthrough just incase that they don't have `oc` installed. The idea is that they should be able to do everything within the browser.

Reply:
ah.. Very valid point!